RELEASING.md

```diff
@@ -0,0 +1,82 @@
+# Apache SkyWalking Nginx LUA agent release guide
+
+SkyWalking Nginx LUA agent released through source tar on the [website](https://skywalking.apache.org/downloads/#NginxLUAAgent) 
+and [SkyWalking's LuaRock Module](https://luarocks.org/modules/apache-skywalking/skywalking-nginx-lua).
+
+Release manager could follow this doc to build and upload a release for this agent.
+
+1. Prepare the new `.rockspec` file for the new release. Ref previous and latest files in [here](./rockspec/).
+2. Update the [changelogs](CHANGES.md) for the upcoming release.
+3. Tag through git or GitHub.
+4. Build the source tars with ASC sign and SHA512
+
+```shell
+> export RELEASE_VERSION=x.y.z
+
+> tar czf skywalking-nginx-lua-${RELEASE_VERSION}-src.tgz \
```

Comment:
There is already a [release.md](https://github.com/apache/skywalking-nginx-lua/blob/master/release.md) file, Do we recommend using `make release-src` here ?

Reply:
Oops, I missed that. I am going to merge these two.